src/monosynth.js

```diff
@@ -77,14 +77,21 @@ class MonoSynth extends AudioVoice {
      * @for p5.MonoSynth
      */
     /**
+     * allow user to set the decay time of the envelope (ADSR) of the MonoSynth class.
+     * It is a getter and setter that can be used to retrieve or change the decay time.
+     * used in conjunction with the attack, sustain, and release fields/functions to set the full envelope of the synthesizer.
```

Comment:
For here and elsewhere in these docs, we should probably be consistent and capitalize the starts of sentences